kml/display-kml/src/main/java/com/esri/samples/display_kml/DisplayKMLSample.java

```diff
@@ -58,12 +59,16 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
       // create a progress indicator
       ProgressIndicator progressIndicator = new ProgressIndicator();
       progressIndicator.setVisible(false);
 
       // create a map and add it to the map view
-      ArcGISMap map = new ArcGISMap(Basemap.createDarkGrayCanvasVector());
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_DARK_GRAY);
       mapView = new MapView();
       mapView.setMap(map);
```

Comment:
Maybe we should split this up (like in most samples), or rephrase the comment so that it's more accurate about what the code does?

` // create a map and add it to a new map view ` would make more sense, beause `the map view` doesn't exist until we create it in this block